web/static/js/components/surveys/SurveyWizardScheduleStep.jsx

```diff
@@ -2,10 +2,10 @@ import * as actions from '../../actions/survey'
 import * as uiActions from '../../actions/ui'
 import { connect } from 'react-redux'
 import React, { PropTypes, Component } from 'react'
-import TimezoneDropdown from '../timezones/TimezoneDropdown'
```

Comment:
`TimezoneDropdown` is not used anymore on the project, so let's delete it's source file too - and every other dependency that's not used anymore, if any.

Should we want to use it again in the future, we'll always have `git`.